src/index.json

```diff
@@ -275,6 +275,51 @@
                     "version": "0.4.37"
                 },
                 "sha256Digest": "80d04f0d477b987dd462e6c1fb3fdbe7441c7ea1e3c75fb8593847d43b4a1e3b"
+            },
+            {
+                "downloadUrl": "https://azurecliprod.blob.core.windows.net/cli-extensions/aks_preview-0.4.38-py2.py3-none-any.whl",
+                "filename": "aks_preview-0.4.38-py2.py3-none-any.whl",
+                "metadata": {
+                    "azext.isPreview": true,
+                    "azext.minCliCoreVersion": "2.0.49",
+                    "classifiers": [
+                        "Development Status :: 4 - Beta",
+                        "Intended Audience :: Developers",
+                        "Intended Audience :: System Administrators",
+                        "Programming Language :: Python",
+                        "Programming Language :: Python :: 2",
+                        "Programming Language :: Python :: 2.7",
+                        "Programming Language :: Python :: 3",
+                        "Programming Language :: Python :: 3.4",
+                        "Programming Language :: Python :: 3.5",
+                        "Programming Language :: Python :: 3.6",
+                        "License :: OSI Approved :: MIT License"
+                    ],
+                    "extensions": {
+                        "python.details": {
+                            "contacts": [
+                                {
+                                    "email": "azpycli@microsoft.com",
```

Comment:
wrong contact point. hi @haroldrandom  is this also auto-generated? how to fix it?

Reply:
We could discard this PR and fire another PR to remove Python 2 from variable `CLASSIFIERS` in setup.py, also change the property `author_email` in setup.py, and upgrade the patch version like `0.4.39`.
Then merge it to wait for another PR for updating index.json